indexing-hadoop/src/main/java/io/druid/indexer/HadoopDruidIndexerConfig.java

```diff
@@ -213,8 +213,8 @@ public static HadoopDruidIndexerConfig fromConfiguration(Configuration conf)
     return retVal;
   }
 
-  private volatile HadoopIngestionSpec schema;
```

Comment:
@b-slim If objects of `HadoopDruidIndexerConfig` are ever transmitted between Threads (what I doubt, someone who knows `hadoop-indexing` module could help here, but I cannot quickly answer this question myself), tracking how `HadoopDruidIndexerConfig` is published shows that it is either stored into a `final` field (that gives the same publication guarantees as `volatile`) or constructed from some configuration or String into a non-final field in `setup()` methods of Reducers, that is not even transmission between threads, because Config didn't come from somewhere, it is constructed right here.